.ci-config/rippled.cfg

```diff
@@ -178,8 +178,8 @@ PriceOracle
 fixEmptyDID
 fixXChainRewardRounding
 fixPreviousTxnID
-# 2.3.0-rc1 Amendments
 fixAMMv1_1
+# 2.3.0 Amendments
 Credentials
```

Comment:
fixAMMv1_2 should be included because it's part of 2.3.0 release - https://xrpl.org/blog/2024/rippled-2.3.0
```suggestion
# 2.3.0 Amendments
fixAMMv1_2
Credentials
```